recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
         - def_snprintf.patch  # [win]
 
 build:
-    number: 3
+    number: 1
```

Comment:
Actually, the more I think about it, maybe we are doing something special with having the exact same build - essentially conda's index concept only allows one package to have a specific name - if we are clobbering the package which comes from defaults, it can't possibly be chosen, so conda resolves differently to if we hadn't clobbered it.